core/profiles/standard/standard.install

```diff
@@ -108,6 +108,42 @@ function standard_install() {
   theme_enable(array($default_theme));
   theme_disable(array('stark'));
 
+  // Enables the title_combo block and positions breadcrumbs
+  // immediately above title_combo block.
+
+  $title_combo_uuid = 'a549d0bc-51f7-46b0-9b41-653788410dd4';
+
+  $title_combo_block = array(
+    'content.' . $title_combo_uuid => array(
+      'plugin' => 'system:page_components:title_combo',
+      'data' => array(
+        'module' => 'system',
+        'delta' => 'page_components',
+        'settings' => array(
+          'title_display' => 'default',
+          'title' => '',
+          'style' => 'default',
+          'block_settings' => [],
+          'contexts' => [],
+          'title_tag' => 'h1',
+          'title_classes' => 'page-title',
+          'tab-type' => 'both',
+        ),
+        'uuid' => $title_combo_uuid,
+        'style' => array(
+          'plugin' => 'default',
+          'data' => array(
+            'settings' => array(
+              'classes' => '',
+            ),
+          ),
+        ),
+      ),
+    ),
+  );
+  config_set_multiple('layout.layout.default', $title_combo_block );
+  config_set('layout.layout.default','positions.top', array('09f11b4e-877e-45ab-9000-a4c6254faa04', 'a549d0bc-51f7-46b0-9b41-653788410dd4'));
```

Comment:
Should we be using the variable here instead?

Reply:
I suppose. I went ahead and used a variable for the breadcrumbs block as well. Not sure if that is necessary or helpful. I thought it might help make the code more understandable.